Sources/Sentry/SentrySDK.m

```diff
@@ -103,49 +101,34 @@ + (nullable SentryAppStartMeasurement *)getAppStartMeasurement
     }
 }
 
-/**
- * Not public, only for internal use.
- */
-+ (NSUInteger)startInvocations
-{
-    return startInvocations;
-}
-
-/**
- * Only needed for testing.
- */
-+ (void)setStartInvocations:(NSUInteger)value
-{
-    startInvocations = value;
-}
-
 + (void)startWithOptions:(NSDictionary<NSString *, id> *)optionsDict
 {
     NSError *error = nil;
     SentryOptions *options = [[SentryOptions alloc] initWithDict:optionsDict
                                                 didFailWithError:&error];
     if (nil != error) {
-        [SentryLog logWithMessage:@"Error while initializing the SDK" andLevel:kSentryLevelError];
-        [SentryLog logWithMessage:[NSString stringWithFormat:@"%@", error]
-                         andLevel:kSentryLevelError];
+        [SentryLog
+            logWithMessage:[NSString stringWithFormat:@"Error parsing init options: %@", error]
+                  andLevel:kSentryLevelError];
     } else {
         [SentrySDK startWithOptionsObject:options];
     }
 }
 
 + (void)startWithOptionsObject:(SentryOptions *)options
 {
-    startInvocations++;
-
-    [SentryLog configure:options.debug diagnosticLevel:options.diagnosticLevel];
-    SentryClient *newClient = [[SentryClient alloc] initWithOptions:options];
-    // The Hub needs to be initialized with a client so that closing a session
-    // can happen.
-    [SentrySDK setCurrentHub:[[SentryHub alloc] initWithClient:newClient andScope:nil]];
-    [SentryLog logWithMessage:[NSString stringWithFormat:@"SDK initialized! Version: %@",
-                                        SentryMeta.versionString]
-                     andLevel:kSentryLevelDebug];
-    [SentrySDK installIntegrations];
+    static dispatch_once_t onceToken;
```

Comment:
Good point

Reply:
Oh interesting. Even though it appears `close` was introduced without the intention of calling `start` again afterwards, since it's possible, we shouldn't break it. Seems like we shouldn't make these changes then.

Reply:
That was maybe the reason why I didn't go for `dispatch_once` 😁. If you can't make it work with `dispatch_once` you add this info as a comment to the counter, so we know it the next time. Sorry about that.

Reply:
No problem, I can do that.

Reply:
@philipphofmann done!